TUTORIAL.md

```diff
@@ -0,0 +1,208 @@
+##Guide to Solving and Reviewing Rails Blogs Nested Forms
+
+###Objective
+- Submit a form with nested attributes.
+- Identify the relationship between `form_for` and `fields_for` class methods.
+
+
+###Step 1: Set up your app
+When working with an existing codebase, it's always a good idea to see what our app looks like in our browser. To do this we need to run a few commands in our console.
+
+- `bundle install` Get all our gems in order
+- `bin/rake db:migrate RAILS_ENV=test` Setup the DB
+- `rails s` then visit `localhost:3000` Start our server
+- `rake db:seed` Seed our database
+
+
+If you visit `http://localhost:3000/posts/new`, you can see we have a form that will let you create a post, along with tags that we can select. Let's go ahead and submit a post to make sure it works. It does, great!
+
+So we have a huge selection of tags, but what if we want to add a new one?
+<img src="https://github.com/learn-co-curriculum/rails-blog-nested-forms/blob/ea837ba87a44f7cb39e8d10233dcd68984f9b40a/app/assets/images/nobuild.jpg?raw=true" width="50%">
+
+###Step 2: Set up nested attributes
+The first thing we want to do is set up our `Post` model so it can accept our nested attributes, which will be tags. Rails gives us a method called `#accepts_nested_attributes_for`. "Nested attributes allow you to save attributes on associated records through the parent. By default nested attribute updating is turned off and you can enable it using the `#accepts_nested_attributes_for` class method. When you enable nested attributes an attribute writer is defined on the model."
```

Comment:
One other thing is before going straight into accepts_nested_attributes, maybe show a bit of the problem it's solving? That like instead of submitting 2 forms, one to submit the Post, and one to submit the tag after you create the post, you can just do it one swoop